lib/Templates.jsx

```diff
@@ -104,7 +104,7 @@ export default (function () {
         for (let argumentNumber = 0; argumentNumber < wantedNamespace.length; argumentNumber++) {
             currentArgument = wantedNamespace[argumentNumber];
 
-            if (_.isUndefined(namespace[currentArgument])) {
+            if (!namespace[currentArgument]) {
```

Comment:
This logic isn't the same, now all falsy values (0, '', null) will result in true. How about we check with `X === undefined`

Reply:
And in all other places

Reply:
I checked other places and changed conditions in some places where it was needed